model_analyzer/record/perf_throughput.py

```diff
@@ -39,6 +38,10 @@ def __init__(self, perf_output, timestamp=0):
             # Get first word after Throughput
             if 'Throughput:' in line:
                 throughput = float(line.split()[1])
+                break
+        else:
```

Comment:
There is an indentation error here I think.

Reply:
No, this is the `for else` block. If the break does not happen, else will be executed.

Reply:
Ah i see, I've never used this syntax before. Thanks!